src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -128,6 +183,7 @@ class LogicActionPolymorphicSerializer(PolymorphicSerializer):
         str(
             LogicActionTypes.set_registration_backend
         ): LogicSetRegistrationBackendActionSerializer,
+        str(LogicActionTypes.synchronize_children): SynchronizeChildrenActionSerializer,
```

Comment:
this might also be made more generic (at least in the backend) and for now we only expose it in the frontend for children.

I can see this being used to synchronize an edit grid from an earlier step into an editgrid for a later step - I know in particular that DH would like to use editgrids to request basic information and then for each item add the ability to add file uploads, this could definitely be iterated on to make that possible in the future.